src/components/home/sections/Volunteer.js

```diff
@@ -29,15 +32,37 @@ export const Volunteer = () => {
                 </div>
 
                 <div className="image-col">
-                    <div className="image-row">
+                    <div className="overlay">
                         <img src={kim} alt="board-img" />
+                        <div className='text-div'>
+                            <p>Amber Kim</p>
+                            <p>Chair</p>
+                        </div>
+                    </div>
+                    <div className="overlay">
+                        <img src={greenCrop} alt="board-img" />
+                        <div className='text-div'>
+                            <p>Cody Green</p>
+                            <p>Vice Chair</p>
+                        </div>
+                    </div>
+                    <div className="overlay">
                         <img src={johnson} alt="board-img" />
+                        <div className='text-div'>
+                            <p>Ryan Johnson</p>
+                            <p>Member of the Board</p>
+                        </div>
                     </div>
-                    <div className="image-row">
-                        <img src={green} alt="board-img" />
-                        <img src={timKim} alt="board-img" />
+                    <div className="overlay">
+                        <img src={timKimCrop} alt="board-img" />
+                        <div className='text-div'>
```

Comment:
Looks like the image itself is not square so forcing it into a square squishes him. Check the other picture also.

Go ahead and delete the old files if we're no longer using them. Don't feel like you need to keep files that we are not using.

Reply:
Okay I'll see if I can fix the sizing so all the images stay the same size